kratos/solving_strategies/builder_and_solvers/builder_and_solver.h

```diff
@@ -448,7 +448,7 @@ class BuilderAndSolver
         const bool MoveMesh
         )
     {
-        KRATOS_WARNING("BuilderAndSolver") << "no special implementation available for BuildAndSolveLinearizedOnPreviousIteration. Using normal BuildAndSolve" << std::endl;
+        KRATOS_WARNING("BuilderAndSolver") << "No special implementation available for BuildAndSolveLinearizedOnPreviousIteration. Using normal BuildAndSolve" << std::endl;
```

Comment:
@loumalouomega  thx for making the changes!!

Reply:
Are you sure this warning is even necessary?
I mean, there is no harm in not having this implementation in a derived class except maybe slower convergence

Reply:
I think it is important to know that an option you prescribe is ignored...

actually i think there is a case for throwin an error...

Reply:
Throwing seems like appropriate in this situation